podpac/core/data/rasterio_source.py

```diff
@@ -219,7 +219,7 @@ def get_data_overviews(self, coordinates):
             new_coords = Coordinates.from_geotransform(
                 dataset.transform.to_gdal(), dataset.shape, crs=self.coordinates.crs
             )
-            window,new_coords = self._get_window_coords_slc(self,coordinates,new_coords)
+            window,new_coords = self._get_window_coords(coordinates,new_coords)
```

Comment:
Very weird that this snuck through the last release. This is certainly a necessary hotfix.

Reply:
Yeah I'm not sure how it got through.  I wonder if no test requested a tile that covered this.  But also, it does return a tile even with this bug, it's just a transparent tile.  We could probably increase the coverage of the unit tests such that they would have caught this error as well.

Reply:
We [HAVE a test](https://github.com/creare-com/podpac/blob/develop-3.X/podpac/core/data/test/test_rasterio.py#L87) that specifically calls this function. It may be that the unit test engine missed it? Very weird.

Reply:
Ah. Yes. That makes perfect sense. I missed that. Thanks for solving that mystery for me.